### PR TITLE
Cleanup custom kernel_module_disabled scenarios

### PR DESCRIPTION
**Pull request by vojtapolasek**
#### Description:

- create test scenarios for kernel_module_disabled template
- remove custom test scenarios

#### Rationale:

cleanup

### CODE REVIEW

File: shared/templates/kernel_module_disabled/tests/file_not_there.fail.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+rm -f /etc/modprobe.d/{{{ KERNMODULE }}}.conf
```
Comment (yuumasato) [inline shared/templates/kernel_module_disabled/tests/file_not_there.fail.sh:3]: This test should probably go through all of the directories checked, not just `/etc/modprobe.d`.

And looking at the check, it looks for any `*.conf` file with `install {{{ KERNMODULE }}}`.
So the test should remove any `*.conf` files in any of the directories checked with `install {{{ KERNMODULE }}}`.

File: linux_os/guide/system/network/network-uncommon/kernel_module_firewire-core_disabled/tests/kernel_module_preparation.sh [hunk 2]
```diff
@@ -1,24 +0,0 @@
...
-        local MODULE=$1
-
-        # Paths to configuration files and directories for modprobe
-        kernel_mod_files=( "/etc/modprobe.d/*.conf" "/etc/modprobe.conf" "/etc/modules-load.d/*.conf" "/run/modules-load.d/.*conf" "/usr/lib/modules-load.d/*.conf" "/run/modprobe.d/*.conf" "/usr/lib/modprobe.d/*.conf" )
-
-        for filename in "${kernel_mod_files[@]}"; do
-                for file in $filename; do
-                        sed -i "/install[[:space:]]+$kern_module/d" $file
-                done
-        done
```
Comment (yuumasato) [inline linux_os/guide/system/network/network-uncommon/kernel_module_firewire-core_disabled/tests/kernel_module_preparation.sh:23]: This snippet can be reused in the test scenarios to handle all the directories the OVAL checks.

File: shared/templates/kernel_module_disabled/tests/file_not_there.fail.sh (hunk 1 above)
Comment (yuumasato) [reply shared/templates/kernel_module_disabled/tests/file_not_there.fail.sh:3]: The `kernel_module_preparation.sh` has some code that can be reused.

File: shared/templates/kernel_module_disabled/tests/comment.fail.sh [hunk 3]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+if [ -f /etc/modprobe.d/{{{ KERNMODULE }}}.conf ]; then
+    sed -i '/install cramfs/d' /etc/modprobe.d/{{{ KERNMODULE }}}.conf
+fi
```
Comment (yuumasato) [inline shared/templates/kernel_module_disabled/tests/comment.fail.sh:5]: This should go through all checked directories.

**Review by yuumasato — CHANGES_REQUESTED** [review]

**Comment by vojtapolasek** [comment]
I think I addressed your feedback. Tests now change all relevant directories.

**Review by yuumasato — APPROVED** [review]

**Pull request merged**